### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by roytman**
Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,7 +78,7 @@ docker-build: build
 	docker build . -t ovsdb-etcd -f dist/images/Dockerfile.ovsdb-etcd
 
 .PHONY: docker-push
-docker-push:
+docker-push: docker-build
```
Comment (hunchback) [inline Makefile:81]: no ... this is intentional ... to enable independent running and debugging of each part ... docker-build will run currently even if image is up to date

Comment (roytman) [reply Makefile:81]: OK, I added another common rule

File: Makefile [hunk 2]
```diff
@@ -78,13 +78,17 @@ docker-build: build
 	docker build . -t ovsdb-etcd -f dist/images/Dockerfile.ovsdb-etcd
 
 .PHONY: docker-push
-docker-push:
+docker-push:
```
Comment (hunchback) [inline Makefile:81]: redundant space at the end

File: Makefile [hunk 3]
```diff
@@ -78,13 +78,17 @@ docker-build: build
...
 	@echo "checking for OVSDB_ETCD_REPOSITORY" && [ -n "${OVSDB_ETCD_REPOSITORY}" ]
 	docker tag ovsdb-etcd ${OVSDB_ETCD_REPOSITORY}/ovsdb-etcd
 	docker push ${OVSDB_ETCD_REPOSITORY}/ovsdb-etcd
 	docker tag etcd ${OVSDB_ETCD_REPOSITORY}/etcd
 	docker push ${OVSDB_ETCD_REPOSITORY}/etcd
 
+.PHONY: docker
+docker: docker-build docker-push
+
+
```
Comment (hunchback) [inline Makefile:91]: renove empty line

File: Makefile (hunk 2 above)
Comment (roytman) [reply Makefile]: done

File: Makefile (hunk 3 above)
Comment (roytman) [reply Makefile]: done

**Pull request merged**